### PR TITLE
230-in-drive-install-in-drive ADDED

### PR DESCRIPTION
**Pull request by oldbilldue18**
## Description
wrote installation steps of TCET Linux using a bootable drive

### CODE REVIEW

File: docs/projects/tcet-linux/installation/installation-in-drive.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+Download the latest version from our [**official website**](https://www.tcetlinux.com/downloads) .
+
+<!-- ![TCET LINUX](/docs/projects/tcet-linux/assets/tldown.png) -->
+<!-- Above image routing is incorrect, refer the route below and update all image routes -->
+![TCET LINUX](../assets/tldown.png)
+### Step 2. Prepare the Bootable USB Stick
+
+- You need to create a bootable USB stick from which you will install your new operating system. You can use various tools for this purpose such as Rufus.
+
+<!-- ![TCET LINUX](/docs/projects/tcet-linux/assets/rufus.png) -->
```
Comment (vish-rt) [inline docs/projects/tcet-linux/installation/installation-in-drive.md:21]: Broken route has been commented. Apply the fix from line 16 and remove the commented code.

File: docs/projects/tcet-linux/installation/installation-in-drive.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+sidebar_label: In-Drive Installation Steps
+keywords: [Installation Steps, Installation, Steps, Guide, linux guide ,  bootable drive]
+---
+TCET Linux Installation `element` is a step-by-step guide created by us that will walk you through the process of installing [**TCET Linux**](https://github.com/tcet-opensource/tcet-linux/releases/download/v0.5-beta/checksum)  using a bootable drive on your system.This `Section` in the [**TCET Linux Website**](https://linux.tcetmumbai.in/) helps the user to easily install `TCET Linux` on their Device.
+
+### Step 1. Download the latest version of TCET LINUX
+
+Download the latest version from our [**official website**](https://www.tcetlinux.com/downloads) .
+
+<!-- ![TCET LINUX](/docs/projects/tcet-linux/assets/tldown.png) -->
```
Comment (vish-rt) [inline docs/projects/tcet-linux/installation/installation-in-drive.md:14]: Broken route has been commented. Review the fix from line 16 and remove this line.

File: docs/projects/tcet-linux/installation/installation-in-drive.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+<!-- Above image routing is incorrect, refer the route below and update all image routes -->
+![TCET LINUX](../assets/tldown.png)
+### Step 2. Prepare the Bootable USB Stick
+
+- You need to create a bootable USB stick from which you will install your new operating system. You can use various tools for this purpose such as Rufus.
+
+<!-- ![TCET LINUX](/docs/projects/tcet-linux/assets/rufus.png) -->
+
+- Firstly, you need to download and install [**Rufus software**](https://rufus.ie/) into your Windows PC.
+<!-- ![TCET LINUX](/docs/projects/tcet-linux/assets/rufusdown.png) -->
```
Comment (vish-rt) [inline docs/projects/tcet-linux/installation/installation-in-drive.md:24]: Broken route has been commented. Apply the fix from line 16 and remove the commented code.

File: docs/projects/tcet-linux/installation/installation-in-drive.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+- Then open Rufus in your device and you will get a prompt (saying do you want to allow this application to make changes to your device), click on **YES** .
+- A Rufus window will open . 
+<!-- ![TCET LINUX](/docs/projects/tcet-linux/assets/rufusprom.png) -->
```
Comment (vish-rt) [inline docs/projects/tcet-linux/installation/installation-in-drive.md:28]: Broken route has been commented. Apply the fix from line 16 and remove the commented code.

File: docs/projects/tcet-linux/installation/installation-in-drive.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+- Here, under device you will able to see the drive or usb device that you have plugged in your device port. 
+- Note that you have selected Disk or ISO image under the Boot selection option.
+- Then  click on `SELECT` and then you want to locateyour ISO which is probably going to be inside the downloads floder. Double click on it and as you can see that tcetlinux ISO flie has been selected.
+- You  want to make sure the partition scheme for the bootable usb is MBR and this will work on both BIOS and UEFI systems.
+- Now keeping everything else as default setting , the next thing we wwill do is go ahead and click on start.
+<!-- ![TCET LINUX](/docs/projects/tcet-linux/assets/rufuspromfinal.png) -->
```
Comment (vish-rt) [inline docs/projects/tcet-linux/installation/installation-in-drive.md:34]: Broken route has been commented. Apply the fix from line 16 and remove the commented code.

File: docs/projects/tcet-linux/installation/installation-in-drive.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+- Now after this there will be a menu box opend which will ask you wether you want to write an ISO image or write the same thing in DD mmode. So what you want to do is select DD mode , this is because in the first option the in ISO image a lot of distribution can't boot in UEFI/BIOS mode , thus DD mode is recommended.
+<!-- ![TCET LINUX](/docs/projects/tcet-linux/assets/isofinal.png) -->
```
Comment (vish-rt) [inline docs/projects/tcet-linux/installation/installation-in-drive.md:36]: Broken route has been commented. Apply the fix from line 16 and remove the commented code.

File: docs/projects/tcet-linux/installation/installation-in-drive.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+- NOw click on OK and a pop up menu will openn which will waarn you that all the data on the usb device is going to be wipped off, so this is where you want to make sure that you have already backed up all the data.
+<!-- ![TCET LINUX](/docs/projects/tcet-linux/assets/rufusdone.png) -->
```
Comment (vish-rt) [inline docs/projects/tcet-linux/installation/installation-in-drive.md:38]: Broken route has been commented. Apply the fix from line 16 and remove the commented code.

**Review by vish-rt — CHANGES_REQUESTED** [review]
Requested some changes in `installation-in-drive.md` kindly have a look into it. Alongside that, more changes to be done are as follows.

- [ ] Remove the `notes` folder. This is Docusaurus default and not supposed to be on production.
- [ ] Remove `package-lock.json` file from your branch.
- [ ] `package.json` should not have any changes in dependencies. Remove any modifications.

**Review by vish-rt — CHANGES_REQUESTED** [review]
Changes to be done are as follows.

- [ ] Remove the `notes` folder. This is Docusaurus default and not supposed to be on production.
- [ ] Remove `package-lock.json` file from your branch.
- [ ] `package.json` should not have any changes in dependencies. Remove any modifications.

**Comment by Akash6222** [comment]
> Changes to be done are as follows.
> 
> * [ ]  Remove the `notes` folder. This is Docusaurus default and not supposed to be on production.
> * [ ]  Remove `package-lock.json` file from your branch.
> * [ ]  `package.json` should not have any changes in dependencies. Remove any modifications.

@oldbilldue18 plz do the required changes

**Pull request closed without merging**